public/app/core/components/TraceToLogs/TraceToLogsSettings.tsx

```diff
@@ -124,14 +134,24 @@ export function TraceToLogsSettings({ options, onOptionsChange }: Props) {
       </InlineFieldRow>
 
       <TimeRangeShift
-        type={'start'}
+        label={getTimeShiftLabel('start')}
+        tooltip={getTimeShiftTooltip('start')}
         value={traceToLogs.spanStartTimeShift || ''}
-        onChange={(val) => updateTracesToLogs({ spanStartTimeShift: val })}
+        onChange={(val) => {
+          setSpanStartTimeShiftIsInvalid(validateTimeShift(val));
```

Comment:
I am not sure I get this, why isn't the `TimeRangeShift` already doing the validation?